README.md

```diff
@@ -1,2 +1,38 @@
 # CityApp-ANET
+
 A simple web application that displays a list of cities
+
+## Prerequisites
+
+* [Docker](https://www.docker.com/)
+* [Docker Compose](https://docs.docker.com/compose/)
+* Run all containers with `docker compose up`
+
+1. Database will run on localhost:5432. 
+1. Frontend will run on http://localhost:5083/api.
+1. Backend  will run on http://localhost:8080
+
+
+## Tech stack
+
+* Database: [PostgreSQL](https://www.postgresql.org/)
+* Backend:  [.NET](https://dotnet.microsoft.com)
+* Frontend: [Angular](https://github.com/angular)
+
+
+## Teamwork notes
+
+* Team setup: Marko (BE, FE), Kristjan (FE), Merily (DevOps, Design),
+* Worked hours: 
+  
+* Benefits of working in a team:
+1. More efficient planning for all parts
+1. Dividing tasks means better focus, faster development and richer implementation
+1. `````fill here smth`````
```

Comment:
The note section below could also be considered as a benefit in my opionin - work doesn't stall when one member falls ill

Reply:
Marko will add this and other notes to README after this branch is finished and merged